c2cgeoform/views/abstract_views.py

```diff
@@ -336,9 +340,17 @@ def edit(self):
         dict_ = form.schema.dictify(obj)
         if self._is_new():
             dict_.update(self._request.GET)
-        rendered = form.render(dict_,
-                               request=self._request,
-                               actions=self._item_actions(obj))
+        if 'msg_col' in self._request.params.keys() and self._request.params['msg_col'] in MSG_COL.keys():
+            rendered = form.render(
+                dict_,
+                request=self._request,
+                actions=self._item_actions(obj),
+                msg_col=[MSG_COL[self._request.params['msg_col']]])
+        else:
+            rendered = form.render(
+                dict_,
+                request=self._request,
+                actions=self._item_actions(obj))
```

Comment:
Is it possible to create a variable `msg_col` instead of duplicating the call to form.render.

Reply:
I try to, but when setting 'msg_col = None', exists at chameleon side is true !